OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -826,6 +827,8 @@ public void Tick(World world)
 				ticksToNextSmoke = info.TrailInterval;
 			}
 
+			world.ScreenMap.Update(this, pos, anim.Image.Bounds);
```

Comment:
`anim` can be `null`, so you should check for that, like in `Bullet` and `GravityBomb`.